### ISSUE TITLE
Declare support with webonyx/graphql-php version 15

### ISSUE THREAD

**Issue by hostep** [hyva-themes/magento2-graphql-view-model#7]
Hi there

I noticed this in the composer output while trying out an upgrade from Magento 2.4.5-p1 to 2.4.6-beta8:

```
- Downgrading hyva-themes/magento2-graphql-view-model (1.0.4 => 1.0.3)
```

After some research, this is because:
1. Certain Magento modules now have the constraint `"^15.0"` on `webonyx/graphql-php`
2. Version 1.0.4 of this module here, [introduced a constraint](https://github.com/hyva-themes/magento2-graphql-view-model/commit/aba9ebdb5aebad94868b951d664711b5fbc0bbbb), so composer downgrades it to 1.0.3 where the constraint didn't exist yet

It's probably a good idea to declare support for both versions in this module (if version 15 works well with the current codebase, which I haven't tested, here's the [changelog](https://github.com/webonyx/graphql-php/blob/master/CHANGELOG.md#v1500) ...): `"webonyx/graphql-php": "^14.0.0 || ^15.0.0"`

Thanks!

**Comment by Vinai** [comment]
Thanks for opening the issue @hostep!
The constraint was introduced because of a missing type cast in graphql-php: https://github.com/hyva-themes/magento2-graphql-view-model/issues/5
Given the history, I think this module probably will require some testing to ensure version 15 actually works.
I want to get back to this before the next Hyvä Themes release.

**Comment by Vinai** [comment]
@hostep `webonyx/graphql-php` v15.x compatibility released in 1.0.5.

**Comment by hostep** [comment]
Awesome, thanks! 🙂
